core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -787,13 +787,24 @@ public final String getUserClaimValue(String userName, String claim, String prof
         if (index > 0) {
             String names[] = claimValue.split(CarbonConstants.DOMAIN_SEPARATOR);
             extractedDomain = names[0].trim();
+        }
+
+        UserStoreManager userManager;
+        if (StringUtils.isNotEmpty(extractedDomain)) {
+            userManager = getSecondaryUserStoreManager(extractedDomain);
+            if (log.isDebugEnabled()) {
+                log.debug("Domain: " + extractedDomain + " is passed with the claim and user store manager is loaded" +
+                        " for give domain name.");
```

Comment:
Small grammar issue with the log message -> It should be "user store manager is loaded for the given domain name."